koala/ast/graph.py

```
@@ -202,35 +246,28 @@ def plot_graph(self):
         nx.draw_networkx_labels(self.G, pos)
         plt.show()
     
-    def set_value(self,address,val,is_addr=True):
-        #
-        if address in self.ranges:
-            # if is_range(address):
-            #     addresses = resolve_range(self.named_ranges[address])[0]
-            #     cell_to_set = []
-            #     for address in addresses:
-            #         if address in self.cellmap:
-            #             cell_to_set += [self.cellmap[address]]
-            #         else:
-            #             print "===", address
-            if type(self.ranges[address]) == Range:
-                cell_to_set = [self.cellmap[a] for a in self.ranges[address].cells if a in self.cellmap]
+    def set_value(self, address, val):
+
+        # case where the address refers to a range
+
+        if type(self.cellmap[address].value) == Range:
+            cell_to_set = [self.cellmap[a] for a in self.cellmap[address].value.cells if a in self.cellmap]
```

Maybe we can raise an exception if address isn't in cellmap